.github/workflows/lint-build-test.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Test
+
+# run CI on pushes to master, and on all PRs (even the ones that target other
+# branches)
+
+on:
+ push:
+   branches: [master]
+ pull_request:
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['12.x', '15.x']
```

Comment:
15.x?  Should this just be 14.x?

Reply:
Current is 15.6.0, so I think at least `nat` and probably `agoric-sdk` as well should test it.